src/chttpd/src/chttpd_db.erl

```diff
@@ -569,15 +569,23 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
             send_json(Req, 417, ErrorsJson)
         end;
     false ->
-        case fabric:update_docs(Db, Docs, [replicated_changes|Options]) of
-        {ok, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 201, ErrorsJson);
-        {accepted, Errors} ->
-            chttpd_stats:incr_writes(length(Docs)),
-            ErrorsJson = lists:map(fun update_doc_result_to_json/1, Errors),
-            send_json(Req, 202, ErrorsJson)
+        Docs2 = [Doc || Doc <- Docs, element(3, Doc) =/= {0, []}],
```

Comment:
Silently dropping documents seems a bit worrying. It's worth keeping in mind that `new_edits=false` option is intended intended for replication [mostly](https://docs.couchdb.org/en/stable/api/document/common.html#put--db-docid) : 

 > The ``false`` value is intended for use only by the replicator.

So we want to make sure a broken replicator implementation doesn't just drop documents on the floor and continue on without noticing